internal/storage/dal/church.go

```diff
@@ -0,0 +1,86 @@
+package dal
+
+import (
+	"github.com/google/uuid"
+	"github.com/jsfan/hello-neighbour-api/internal/storage/models"
+	"github.com/jsfan/hello-neighbour-api/pkg"
+)
+
+func (dalInstance *DAL) InsertChurch(churchIn *pkg.ChurchIn) error {
```

Comment:
I usually use `QueryContext` an then put a `RETURNING id` at the end of the query. That way, you get the DB id right with the insert.